### DISCUSSION TITLE
3D Viewer is not working

### DISCUSSION THREAD

**Discussion by ohhh-source-code** [google/model-viewer#4385] [Q&A] [answered]
I must be missing something basic.

After successfully installing everything , 
from chrome browser I have opened the following index.html in  packages/examples/modelviewer.dev/

3D model viewer is apparently not working.
![photo_2023-07-24_01-20-49](https://github.com/google/model-viewer/assets/46641884/e853f580-1fe3-4d17-9451-0c42e3395cdf)

What am I missing?

**Comment by diegoteran** [comment]
Do you have any console log warning/errors you could share? Is this image from a mobile device?

**Reply by ohhh-source-code** [reply]
Yes. That is from my phone. I will re-create this on my desktop chrome.

**Comment by ohhh-source-code** [comment]
![error_0](https://github.com/google/model-viewer/assets/46641884/df9df8ff-0b41-4c72-a4ef-6d41f1200e50)

**Comment by ohhh-source-code** [comment]
Now it is working with index.html as follows
```
<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
<!-- Example 1 -->
<script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.1.1/model-viewer.min.js"></script>
<model-viewer src="https://modelviewer.dev/assets/boom_2_.glb" ar ar-placement="wall" camera-controls touch-action="pan-y" alt="A 3D model of some wall art"></model-viewer>

</body>
</html>
```

**Comment by ohhh-source-code** [comment]
But I have a different problem now. My initial object appears in the corner like this. 
When I try AR and come back to object, it looks fine though.
Only in the first view is weird. 
![error_3](https://github.com/google/model-viewer/assets/46641884/221ef04a-51ee-4a27-83e6-f8f556244bc3)

**Comment by ohhh-source-code** [comment]
![photo_2023-07-26_01-07-42](https://github.com/google/model-viewer/assets/46641884/9b1509e9-c6f9-4c0b-a79b-002208b11f7a)

If I click on AR button, and click "object" button again. I get this full screen.
Is there anyway I can get this, when I initially load the index.html?

**Comment by ohhh-source-code** [comment, marked as answer]
<style>
model-viewer {
  width: 100vw;
  height: 100vh;
}
</style>

THis did the trick!
